app/controllers/RiderController.java

```diff
@@ -59,24 +55,7 @@ public RiderController(RiderRepository riderRepository) {
         });
     }
 
-    @BodyParser.Of(BodyParser.Json.class)
-    public CompletionStage<Result> addRider(){
-        JsonNode json = request().body().asJson();
-        return parseRider(json).thenApply(rider -> riderRepository.addRider(rider)).thenApply(rider -> {
-            return ok(rider + " has been added");
-        }).exceptionally(ex -> {
-            Result res = null;
-            switch (ExceptionUtils.getRootCause(ex).getClass().getSimpleName()){
-                case "NullPointerException":
-                    res = badRequest("json format of rider was wrong");
-                    break;
-                default:
-                    res = internalServerError(ex.getMessage());
-            }
-            return res;
-        });
-    }
-
+    /*
     private CompletableFuture<Rider> parseRider(JsonNode json){
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)